### DISCUSSION TITLE
Why webpack-dev-server should be used for development only？

### DISCUSSION THREAD

**Discussion by mrdulin** [webpack/webpack#18515] [Q&A]
Can you explain it in detail?

**Comment by alexander-akait** [comment]
Because usually you have production server for your application and most of features in dev server are only for development

**Comment by tatsmaki** [comment]
Dev server is designed to serve unoptimised development build, verify code against eslint/prettier rules, reflect code changes. And many more operations that you don't want to have in production.

When your app is facing the world you should use tools like nginx, caddy, traefik. They have much more functionality for production use cases.
